app/services/expired_dossiers_deletion_service.rb

```diff
@@ -138,7 +138,7 @@ def self.delete_expired_and_notify(dossiers_to_remove)
 
   def self.group_by_fonctionnaire_email(dossiers)
     dossiers
-      .with_notifiable_procedure
+      .with_notifiable_procedure(true)
```

Comment:
Pareil, le true n'est pas très clair.

Que dirai-tu de :

```suggestion
      .with_notifiable_procedure(include_closed: true)
```